### PR TITLE
Generalize homepage spec

### PR DESCRIPTION
**Pull request by sneakysneakysneaky**
Update homepage spec to make it more generic. References to data involving music is now an example rather than a requirement.

### CODE REVIEW

File: doc/specs/homepage.md [hunk 1]
```diff
@@ -12,9 +12,11 @@ The homepage is the initial landing page for all authenticated users. It provide
 
 ## Business rules.
 
-The homepage contains the following sections:
+The homepage is a collection of customizable sections. A section is a small block of content, limited in scope. The user may personalize their homepage by adding, removing, & re-ordering sections. Sections that would otherwise appear empty do not appear.
 
-* **Site News**: An announcement containing critical information about the site. This section must appear at the top of the page & may not be removed by the user.
+In addition, a "Site News" section appears at the top. It is used for critical announcmenets for hte site. If the "Site News" section contains content, then the section appears at the top of the page & cannot be removed. Otherwise, the section does not appear.
```
Comment (ghost) [inline doc/specs/homepage.md]: 'hte' is a typo, it's spelled 'teh'.

File: doc/specs/homepage.md [hunk 2]
```diff
@@ -12,9 +12,11 @@ The homepage is the initial landing page for all authenticated users. It provide
...
+
+Some examples of sections a user may want to add to their site might be:
 
 * **Notifications**: A summary of the current user's notifications. Displays a maximum of five notifications at a time. Each notifications details may be expanded/collapsed. A user may hide the notification on this screen. Finally, the user may navigate to the notifications page.
```
Comment (ghost) [inline doc/specs/homepage.md:21]: "maximum of five notifications"

Is this an arbitrary number or is there reasoning behind it?

File: doc/specs/homepage.md (hunk 1 above)
Comment (ghost) [inline doc/specs/homepage.md]: By site news do you mean the module that the user can add/remove from their page or the notification? I think it should be up to the user if they want a "Site news" module or not. Instead unread site news should be presented in a modal window (or similar presentation) when the user signs in. If site news is published while the user is browsing the site then it should be presented as a notification.

Comment (thomwiggers) [inline doc/specs/homepage.md]: also `announcmenets`

File: doc/specs/homepage.md (hunk 2 above)
Comment (sneakysneakysneaky) [inline doc/specs/homepage.md:21]: Is arbitrary. I think was the number of items visible in @chriscubellis's mockup. Should probably be configurable.

**Comment by sneakysneakysneaky** [comment]
@ajax-wcd, the "Site News" section refers to this guy. Will display stuff like outage notices & other critical information. I think should be the one mandatory element on the homepage & cannot be hidden by the user.

00a-index 1

**Comment by ghost** [comment]
@sneakysneakysneaky Ah okay. We're on the same page then I believe.

**Comment by ghost** [comment]
@sneakysneakysneaky Can this be merged?

**Comment by sneakysneakysneaky** [comment]
Squashed to ec9fd321695a1ebdf06883e551dae15210e243da & merged

**Pull request closed without merging**